### ISSUE TITLE
set seems to work but get fails

### ISSUE THREAD

**Issue by yieme** [cayasso/cacheman-mongo#3]
When I do a cache.set('test', 'Test Value')

In mongo I see:

```js
{
    "_id": {
        "$oid": "5549661e89332748a381b58d"
    },
    "key": "cache::test",
    "value": "Test Value",
    "expire": 1430873900201
}
```

When I do a 
```js
cache.get('test, function(err, data) {
   console.log(data, typeof data) // null object
}
```

Thoughts?

**Comment by igorissen** [comment]
Same issue here. The first call to `get` work but when calling it twice I keep getting null object.

**Comment by cayasso** [comment]
@yieme @igorissen I created this test https://github.com/cayasso/cacheman-mongo/blob/master/test/index.js#L168 but cannot reproduce the behaviour you guys describe, can any of you post a failing test so we can start from there?

**Comment by cayasso** [comment]
BTW. I just released cacheman-mongo@1.0.0

**Comment by cayasso** [comment]
Closing this do to no response, feel free to reopen with sending a failing test.
